subsys/bluetooth/controller/hci_internal.c

```diff
@@ -310,6 +310,26 @@ static void supported_commands(sdc_hci_ip_supported_commands_t *cmds)
 #endif
 }
 
+#if defined(CONFIG_BT_HCI_VS)
+static void vs_supported_commands(sdc_hci_vs_zephyr_supported_commands_t *cmds)
```

Comment:
@rugeGerritsen I think this is a problem for regular supported commands / features as well.
For example BT_CTRL_PRIVACY?

Reply:
We have already implemented this for normal commands: https://github.com/nrfconnect/sdk-nrf/blob/master/subsys/bluetooth/controller/hci_internal.c#L156

Reply:
@jori-nordic , do you mind adding a commit doing the same thing for "HCI Read Supported Features"?

Reply:
yep no problem, sdk-nrf is kinda blocked for merges with the ongoing upmerge anyways :)

Reply:
Sorry, I mixed it up with "LE Read Local Supported Features command". That one is more important

Reply:
and requires a bit more work :), I'll look into it